src/renderer.cpp

```diff
@@ -701,7 +958,7 @@ void Renderer::draw()
     }
     bgfx::setState(internalState.state);
 
-    bgfx::submit(backendViewId, backendProgramHandles[backendProgram]);
+    bgfx::submit(VIEW_ID_POSTEFFECT, backendProgramHandles[backendProgram]);

```

Comment:
Also here, the idea of having the same var in each stage means we have a progressive view when dumping using RenderDoc. Maybe I'm missing something here, but I'm open for clarifications :) Thanks

Reply:
Please see the comment above but other than the blit command problem, the other view ids are still progressive.
The shadow map pass rendering is view id  0, normal scene rendering is 1, and posteffect is 2.

Reply:
As explained in the previous comment, I reverted to using the `backendViewId` variable.